cmd/podman/commit.go

```diff
@@ -49,15 +49,20 @@ var (
 	 set the commit message with the --message flag,
 	 and make changes to the instructions with the --change flag.`
 	commitCommand = cli.Command{
-		Name:        "commit",
-		Usage:       "Create new image based on the changed container",
-		Description: commitDescription,
-		Flags:       commitFlags,
-		Action:      commitCmd,
-		ArgsUsage:   "CONTAINER [REPOSITORY[:TAG]]",
+		Name:         "commit",
+		Usage:        "Create new image based on the changed container",
+		Description:  commitDescription,
+		Flags:        commitFlags,
+		Action:       commitCmd,
+		ArgsUsage:    "CONTAINER [REPOSITORY[:TAG]]",
+		OnUsageError: usageErrorHandler,
 	}
 )
 
+func usageErrorHandler(context *cli.Context, err error, isSubcommand bool) error {
```

Comment:
Maybe move this over to common.go?

Reply:
That's where I intended to move it ... eww, thanks!